.pre-commit-config.yaml

```diff
@@ -28,14 +28,9 @@ repos:
     rev: "v21.1.1"
     hooks:
       - id: clang-format
-        # Explicitly include ".txx" files, which are not recognized as source files
-        # by the "identify" package used by pre-commit. To work around this,
-        # we manually specify file matching via the "files" regex and set "types_or"
-        # to allow formatting of files treated as plain text.
-        #
         # Reference: https://github.com/pre-commit/identify/blob/main/identify/extensions.py
         types_or: [file, text]
-        files: \.(cpp|cxx|h|hpp|txx)$
+        files: \.(cpp|cxx|h|hpp|hxx)$
```

Comment:
A number of Slicer extensions, VTK, ITK (yes, ITK - via GDCM), Plastimatch, etc. all use .txx, so I don't think we are quite ready to remove txx from here yet.

Reply:
Since this pre-commit-config is only for Slicer core code, this change would seem appropriate. I don't think Slicer extensions, VTK, ITK, etc are following the Slicer pre-commit-config exactly.

Reply:
If Slicer core no longer uses txx files than it would make sense for the pre-commit-config to reflect that because then a Slicer extension developer would also make sure to match the Slicer core usage of txx instead of hxx if they are  matching the pre-commit-config.